tesla_ble_mqtt/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
 
+## 0.0.11
+
+### Changed
+
+- No need to provide MAC addresses anymore for presence detection
+- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
+- Improved presence detection
+- Improved logging (bashio timestamps, verbosity, colors)
+- Added "debug" entity which sends only one charge amps command: https://github.com/tesla-local-control/tesla_ble_mqtt_core/issues/19
+- Clarified issues with BLE device overheating causing performance issues: https://github.com/tesla-local-control/tesla-local-control-addon/issues/27
```

Comment:
We can have this but long term I think this should be in a Troubleshooting section or readme ?!

Reply:
See https://github.com/tesla-local-control/tesla-local-control-addon/pull/62 --> Will merge to this PR.